gprofiler/profilers/python.py

```diff
@@ -355,6 +422,7 @@ def __init__(
         stop_event: Event,
         storage_dir: str,
         python_mode: str,
+        python_add_versions: bool,
```

Comment:
IMO
```suggestion
        python_add_package_versions: bool,
```
is better

Reply:
I chose this name because it's not necessarily packages but also built-in modules. I think this name is more accurate.

Reply:
meh ok